Framework/Algorithms/test/HRPDSlabCanAbsorptionTest.h

```diff
@@ -107,9 +107,9 @@ class HRPDSlabCanAbsorptionTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(result->y(1).front(), 0.7033, 0.0001);
     TS_ASSERT_DELTA(result->y(1)[5], 0.5939, 0.0001);
     TS_ASSERT_DELTA(result->y(1).back(), 0.5192, 0.0001);
-    TS_ASSERT_DELTA(result->y(2).front(), 0.7337, 0.0001);
-    TS_ASSERT_DELTA(result->y(2)[5], 0.6404, 0.0001);
-    TS_ASSERT_DELTA(result->y(2).back(), 0.5741, 0.0001);
+    TS_ASSERT_DELTA(result->y(2).front(), 0.7332, 0.0001);
```

Comment:
Just a quick question after I'd had a second thought. As you say I think the fact that many system tests have not broken is a sign this is probably fine. The part I don't understand is why these values changed at all as the correction for HRPD basically just runs FlatPlateAbsorption and then just puts an analytical correction on the results. It implies `totalDistInsideObject` is returning a different answer in this case?

Reply:
Yes... Some basic tests for this method were added for FlatPlate in #29601 I think the reason there is a difference is captured in the comment I removed https://github.com/mantidproject/mantid/blob/cfed1a6d87ee90f2be7d90db85c303ce35b7c44f/Framework/Algorithms/src/AbsorptionCorrection.cpp#L440-L448

Reply:
But maybe @ckendrick has a better explanation.

Reply:
@martyngigg So it looks like the difference could be down to duplicate Tracks being created for particular points and therefore the distance being counted twice for those points by totalDistInsideObject. This is currently being sorted out in #29599 so I will wait for that to go through and see what difference that makes.

Reply:
I think I fixed an issue in #29599 so that this test should pass without changing these three test references slightly, but I still need to confirm if this "fix" causes correct behavior or not.

Reply:
@rosswhitfield @ckendrick Ah yes I understand now. `AbsorptionCorrection` was originally written, I think, to only consider solid shapes so taking just the first link was fine but that did paper over this instability. The fix in #29599 *looks* like the correct thing to do and I guess a good test would be whether these tests stay the same. Nice work tracking it down @ckendrick.